### ISSUE TITLE
Permission error after shrinking

### ISSUE THREAD

**Issue by Daniel-Fan** [IBM/operand-deployment-lifecycle-manager#584]
After the PR of shrinking the permission of ODLM CRDs,

Get permission error:
```
E0101 07:44:15.147649       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1alpha1.OperandRegistry: operandregistries.operator.ibm.com is forbidden: User "system:serviceaccount:common-service-installed-2:operand-deployment-lifecycle-manager" cannot list resource "operandregistries" in API group "operator.ibm.com" at the cluster scope
E0101 07:44:43.711948       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1alpha1.OperandBindInfo: operandbindinfos.operator.ibm.com is forbidden: User "system:serviceaccount:common-service-installed-2:operand-deployment-lifecycle-manager" cannot list resource "operandbindinfos" in API group "operator.ibm.com" at the cluster scope
E0101 07:44:52.259839       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1alpha1.OperandConfig: operandconfigs.operator.ibm.com is forbidden: User "system:serviceaccount:common-service-installed-2:operand-deployment-lifecycle-manager" cannot list resource "operandconfigs" in API group "operator.ibm.com" at the cluster scope
```

/cc @horis233

**Comment by horis233** [comment]
Nice catch @Daniel-Fan, Please open a PR to add the permissions back.

I will think of a method to enhance our current authorization logic.

**Comment by horis233** [comment]
Fixed by #585
